common/pkcs11.h

```diff
@@ -428,6 +446,7 @@ typedef unsigned long ck_attribute_type_t;
 #define CKA_EXPONENT_1			(0x126UL)
 #define CKA_EXPONENT_2			(0x127UL)
 #define CKA_COEFFICIENT			(0x128UL)
+#define CKA_PUBLIC_KEY_INFO		(0x129UL)
```

Comment:
This is also defined in pkcs11x.h. It makes sense to have it here, but we may need to protect it in case a file includes both headers.

Reply:
pkcs11x.h is wrapping its definition with `#ifndef CKA_PUBLIC_KEY_INFO`. In files where both `pkcs11.h` and `pkcs11x.h` are included, the order is to have `pkcs11.h` first.